Packages/com.unity.inputsystem/InputSystem/Actions/InputActionSetupExtensions.cs

```diff
@@ -502,6 +497,19 @@ private static int AddBindingInternal(InputActionMap map, InputBinding binding,
             return bindingIndex;
         }
 
+        private static void InvalidateActionMap(InputActionMap map)
```

Comment:
Name seems a bit odd as it doesn't really invalidate the action map itself. Maybe `MarkActionMapSetupDirty` or something?